README.md

```diff
@@ -1,212 +1,217 @@
-# RHTAS Ansible Collection
+# Red Hat Trusted Artifact Signer Ansible collection
 
-Automation to deploy the RHTAS ecosystem on RHEL
+The purpose of this Ansible collection is to automate the deployment of the Red Hat Trusted Artifact Signer (RHTAS) service on Red Hat Enterprise Linux (RHEL).
 
-:warning: **The contents of this repository are a Work in Progress.**
+> [!WARNING]
+Currently, the contents of this repository are a work in progress and can change often.
 
 ## Overview
 
-The automation within this repository establishes the components of RHTAS, the downstream redistribution of [Sigstore project](https://sigstore.dev) within a single Red Hat Enterprise Linux (RHEL) machine using a standalone containerized deployment. Containers are spawned using Kubernetes based manifests using
-[podman kube play](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
+The RHTAS service is the downstream redistribution of the [Sigstore](https://sigstore.dev) project.
+The automation contained within this Git repository installs and configures the components of RHTAS to run on a single RHEL server, which uses a standalone containerized deployment.
+A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
 
-The following Sigstore components are deployed as part of this architecture:
+The RHTAS Ansible collection deploys the following RHTAS components:
 
 * [Rekor](https://docs.sigstore.dev/rekor/overview)
-    * [Trillian](https://github.com/google/trillian)
-    * Optionally a MariaDB instance and a Redis instance, although it is possible to use instances managed outside of Ansible
+  * [Trillian](https://github.com/google/trillian)
+  * Optional.
+    A self-managed MariaDB instance, and a Redis instance.
 * [Fulcio](https://docs.sigstore.dev/fulcio/overview)
 * [Certificate Log](https://docs.sigstore.dev/fulcio/certificate-issuing-overview)
 * [Timestamp Authority](https://docs.sigstore.dev/verifying/timestamps/#timestamp-authorities)
-* [TUF](https://theupdateframework.io/) server
+* [TUF](https://theupdateframework.io/)
```

Comment:
TUF is a framework, I think the ` server` word is actually important here, because "TUF" itself cannot be deployed

Reply:
Good point!  Updated accordingly.